### ISSUE TITLE
[16.0]l10n_it_delivery_note: creando una fattura da più ddt le righe non vengono generate basandosi sulle righe dei ddt

### ISSUE THREAD

**Issue by odooNextev** [OCA/l10n-italy#3979]
In riferimento alla pr: https://github.com/OCA/l10n-italy/pull/3552
Volendo creare una fattura da più ddt, al momento, la fattura viene generata tenendo conto delle righe dell'ordine di vendita.
In questo modo nelle fatture e nella stampa non vengono generate più righe basandosi sul ddt
Sarebbe utile generare invece una fattura con più righe per ogni riga nei vari ddt:
![image](https://github.com/OCA/l10n-italy/assets/93990941/a7a57dcf-904f-4911-9896-47b3ccd56a42)

**Comment by monen17** [comment]
@odooNextev puoi chiarire in descrizione quali sono le PR che risolvono questa problematica?
A quanto ho capito per `16.0` ci andrebbe https://github.com/OCA/l10n-italy/pull/4385, quindi mi aspettavo che sostituisse #4184, ma vedo che #4184 è stata chiusa perché stale.

**Comment by odooNextev** [comment]
> [@odooNextev](https://github.com/odooNextev) puoi chiarire in descrizione quali sono le PR che risolvono questa problematica? A quanto ho capito per `16.0` ci andrebbe [#4385](https://github.com/OCA/l10n-italy/pull/4385), quindi mi aspettavo che sostituisse [#4184](https://github.com/OCA/l10n-italy/pull/4184), ma vedo che [#4184](https://github.com/OCA/l10n-italy/pull/4184) è stata chiusa perché stale.

Giusto, ho sostituito la https://github.com/OCA/l10n-italy/pull/4184 con la https://github.com/OCA/l10n-italy/pull/4385
